radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/ParticleTransition.java

```diff
@@ -0,0 +1,63 @@
+package com.radixdlt.client.core.ledger;
+
+import com.radixdlt.client.core.atoms.particles.Spin;
+
+/**
+ * Represents an action in the particle state machine. Includes the main instructions
+ * (N2U, U2D) and their reverts respectively (U2N, D2U).
+ */
+public enum ParticleTransition {
+	N2U(Spin.NEUTRAL, Spin.UP),
```

Comment:
Similar comment as the two below, I think `neutralToUp` etc. would be more readable